robots.txt

```diff
@@ -26,3 +36,7 @@ Disallow: /ee/
 Disallow: /apidocs/v*
 
 Sitemap: https://docs.docker.com/sitemap.xml
+{%- else -%}
+User-agent: *
+Disallow: /
+{%- endif -%}
```

Comment:
Not sure if it's checking for that, but looks like the `%-` and `-%` on both sides causes no newline to be present at the end; perhaps (one of them) needs to be removed